src/dmd/e2ir.d

```diff
@@ -1479,8 +1479,8 @@ elem *toElem(Expression e, IRState *irs)
                     break;
 
                 case TYcldouble:
-                    e.EV.Vcldouble.re = re;
-                    e.EV.Vcldouble.im = im;
+                    e.EV.Vcldouble.re = cast(real)re;
+                    e.EV.Vcldouble.im = cast(real)im;
```

Comment:
This cast is no longer necessary and is harmful to the LDC build, too.

Reply:
ok, gone